persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -95,16 +95,18 @@ public static void assertGeneratedSources(String subDir, Command cmd) {
         }
     }
 
-    public static void assertGeneratedSourcesNegative(String subDir, Command cmd, String object) {
+    public static void assertGeneratedSourcesNegative(String subDir, Command cmd, String[] objects) {
         Path sourceDirPath = Paths.get(GENERATED_SOURCES_DIRECTORY, subDir);
         generateSourceCode(sourceDirPath, cmd);
         if (cmd == Command.DB_PUSH) {
             Assert.assertFalse(false);
         } else {
             Assert.assertTrue(directoryContentEquals(Paths.get(RESOURCES_EXPECTED_OUTPUT.toString()).resolve(subDir),
                     Paths.get(GENERATED_SOURCES_DIRECTORY).resolve(subDir)));
-            Path actualObjectFilePath = sourceDirPath.resolve(object);
-            Assert.assertFalse(Files.exists(actualObjectFilePath));
+            for (String object : objects) {
+                Path actualObjectFilePath = sourceDirPath.resolve(object);
+                Assert.assertFalse(Files.exists(actualObjectFilePath));
```

Comment:
```suggestion
            for (String filepath : relativeFilepaths) {
                Path absoluteFilepath = sourceDirPath.resolve(filepath);
                Assert.assertFalse(Files.exists(absoluteFilepath));
```

Reply:
give correct variable name so if code is readable

Reply:
updated in [d1e9e41](https://github.com/ballerina-platform/persist-tools/pull/52/commits/d1e9e4195fff39c00f1fce312aa40487f7196262)